extensions/raycast-quick-search/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "raycast-quick-search",
```

Comment:
```suggestion
  "name": "quick-search",
```

Reply:
The "raycast" is implicit, no need to add it to the extension name (would be redundant in the resulting store URL).